dragonfly/world/mcdb/provider.go

```diff
@@ -85,6 +86,11 @@ func (p *Provider) initDefaultLevelDat() {
 	p.d.MultiPlayerGame = true
 }
 
+// FirstLoad returns true if this is the first time the world was loaded.
+func (p *Provider) FirstLoad() bool {
```

Comment:
The naming of this function breaks the pattern used in the rest of the file. It might be best to come up with a new name which matches the pattern to be consistent.

Reply:
Would "LoadFirstLoad" be a better name?